README.md

```diff
@@ -6,6 +6,7 @@ The workflow to provision AWS AMI is done using pull requests.
 
 ### Requirements
 * Install [packer](https://www.packer.io/intro/getting-started/install.html)
+* pip install ansible
```

Comment:
not needed.  refer to this script would be better, https://github.com/Sage-Bionetworks/ami-provisioner/blob/master/install_packer.sh